can/broadcastmanager.py

```diff
@@ -250,7 +265,11 @@ def _run(self):
                     self.bus.send(self.messages[msg_index])
                 except Exception as exc:
                     log.exception(exc)
-                    break
+                    if self.on_error:
+                        if self.on_error(exc) is True:
```

Comment:
You shouldn't make an explicit comparison with booleans. From PEP-8:

```
Yes:   if greeting:
No:    if greeting == True:
Worse: if greeting is True:
```

Reply:
Got it. Updated to `not ...` as per your other comment.